outputmanager.cpp

```diff
@@ -16,3 +16,11 @@ OutputManager::OutputManager(QObject *parent) : QObject(parent)
 
 }
 
+void OutputManager::handleNewData(const QByteArray &data)
+{
+    // append raw data to the session buffer
+    buffer.append(data);
+
+    // notify that we have new data (in display format)
+    emit dataConverted(QString::fromLocal8Bit(data).remove('\n'));
```

Comment:
At first i didn't realize that the line endings i was sending were also sent back by the serial port. So as i was sending ```\r\n``` as line endings, i was also receiving them. 

I didn't have the same output as the original cutecom. So to solve the *problem*, (that in fact i created myself by sending  ```\r\n``` instead of just  ```\n```: unix endlines), i decided to remove all  ```\n```  of the returned string.
The result on the output was ok, because QTextEdit interprets ```\r\n```, ```\r``` and ```\n``` all in the same way.

Honestamento non ci avevo pensato tanto tempo. Ora faccio un push dove cambio LINE_ENDING, lo metto a ```"\n"```. Vedremo piu tardi se puo servire a l'utente di cambiarla (come su cutecom)
E tolgo ovviamente il ```remove('\n')```